### DISCUSSION TITLE
Place custom markers

### DISCUSSION THREAD

**Discussion by vromaniello** [stella-cv/stella_vslam#262] [Q&A] [answered]
How can I create videos/images to use with OpenVSLAM tracking? From the examples, the videos already have markers positioned inside the place. 

How do you add markers, for example inside my building? 
What characteristics should they have and how can they be mapped offline? 

Thank you

**Comment by ymd-stella** [comment, marked as answer]
Marker integration is an experimental feature. Use only if you understand the code.
